account_financial_risk/tests/test_account_financial_risk.py

```diff
@@ -13,6 +13,8 @@ def setUpClass(cls):
         type_revenue = cls.env.ref('account.data_account_type_revenue')
         type_receivable = cls.env.ref('account.data_account_type_receivable')
         tax_group_taxes = cls.env.ref('account.tax_group_taxes')
+        main_company = cls.env.ref('base.main_company')
+        main_company.currency_id = cls.env.ref('base.USD')
```

Comment:
This triggers a constraint, so it can't be done this way

Reply:
Maybe done with a query?

Reply:
First the convenience of this patch should be determined before "patching" other things